math/cmath/include/algebra/math/algorithms/matrix/inverse/cofactor.hpp

```diff
@@ -0,0 +1,230 @@
+/** Algebra plugins library, part of the ACTS project
+ *
+ * (c) 2022 CERN for the benefit of the ACTS project
+ *
+ * Mozilla Public License Version 2.0
+ */
+
+#pragma once
+
+// Project include(s).
+#include "algebra/qualifiers.hpp"
+
+// System include(s)
+#include <type_traits>
+#include <utility>
+
+namespace algebra::cmath::matrix::inverse {
```

Comment:
I think you have to write your own backend for VC...

Reply:
It should always be possible to get single elements from a Vc type, hence this should *work* (it halfway does right now, if I see it correctly). However, we should do that as little as possible, since it is less efficient, I would expect. So, we should try to avoid calling an ```element_getter``` for Vc wherever we can. But let's leave it as it is and we change it later, should it come to that.